### ISSUE TITLE
LDA DeprecationWarning converting an array with ndim > 0 to an index will result in an error

### ISSUE THREAD

**Issue by tmylk** [RaRe-Technologies/gensim#576]
DeprecationWarning in Python 3 [build](https://travis-ci.org/piskvorky/gensim/jobs/99161647#L86) caused by https://github.com/piskvorky/gensim/blob/develop/gensim/models/ldamodel.py#L874

```
testTopTopics (gensim.test.test_ldamodel.TestLdaModel) ... /home/travis/build/piskvorky/gensim/gensim/models/ldamodel.py:874: DeprecationWarning: converting an array with ndim > 0 to an index will result in an error in the future
  for l in top_words[:m_index - 1]:
ok
```

**Comment by prakhar2b** [comment]
@tmylk 
I want to work on this bug. Could you update me regarding the current status of this bug. Also, the link for **build** seems broken in above comment.

**Comment by piskvorky** [comment]
What's the status here?

**Comment by tmylk** [comment]
This has indeed become an error in numpy 1.12 and was fixed in #1146
